modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -21,6 +21,7 @@ any questions.
   - [Text Input (Preview)](#text-input-preview)
 - [Component Updates](#component-updates)
   - [Component Style Updates](#component-style-updates)
+  - [Component Token Updates](#component-token-updates)
```

Comment:
I wonder if we should break this out so it's not under "component updates" since it's reads like `Card` is under `token-updates`.

```suggestion
- [Component Style Updates](#component-style-updates)
- [Component Token Updates](#component-token-updates)
- [Component Updates](#component-updates)
```